pkgs/misc/emulators/cdemu/analyzer.nix

```diff
@@ -0,0 +1,15 @@
+{ callPackage, gtk3, libxml2, gnuplot, makeWrapper, stdenv }:
+let pkg = import ./base.nix {
+  version = "3.0.0";
+  pkgName = "image-analyzer";
+  pkgSha256 = "1rb3f7c08dxc02zrwrkfvq7qlzlmm0kd2ah1fhxj6ajiyshi8q4v";
+};
+in callPackage pkg {
+  buildInputs = [ gtk3 libxml2 gnuplot (callPackage ./libmirage.nix {}) makeWrapper ];
+  drvParams = {
+    postFixup = ''
+      wrapProgram $out/bin/image-analyzer \
+        --set XDG_DATA_DIRS "$out/share:${gtk3}/share/gsettings-schemas/${gtk3.name}:$XDG_DATA_DIRS"
```

Comment:
Ad. gsettings schema:

When running the image-analyzer program from command line, it failed with `GLib-GIO-ERROR **: Settings schema 'org.gtk.Settings.FileChooser' is not installed`, while using the .desktop file from the gnome launcher was fine.

I found out, that the environment - var in question was `XDG_DATA_DIRS=/nix/store/j1pfpflc5v970c6rvs2qx9dxzzm7cxrj-gtk+3-3.12.2/share/gsettings-schemas/gtk+3-3.12.2`, so I set up the var like you see in analyzer.nix

For the other packages, It sufficed to forward vars from the build environment and maybe I missed some builtin configuration mechanism. I'm not sure how gnome-shell sets up `XDG_DATA_DIRS` and whether nixos can do the same.